tests/utils/parsers/test_parser_options.py

```diff
@@ -0,0 +1,20 @@
+import unittest2
+
+from tuned.utils.parsers.parser_options import ParserOptions
+
+class ParserOptionsTestCase(unittest2.TestCase):
+
```

Comment:
The test case should be added in a single commit along with the ParserOptions class.